pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -355,6 +355,7 @@ func (c *Cluster) cephMgrOrchestratorModuleEnvs() []v1.EnvVar {
 		{Name: "ROOK_OPERATOR_NAMESPACE", Value: operatorNamespace},
 		{Name: "ROOK_CEPH_CLUSTER_CRD_VERSION", Value: cephv1.Version},
 		{Name: "ROOK_CEPH_CLUSTER_CRD_NAME", Value: c.clusterInfo.NamespacedName().Name},
+		{Name: "CEPH_ARGS", Value: fmt.Sprintf("-m $(ROOK_CEPH_MON_HOST) -k %s", keyring.VolumeMount().KeyringFilePath())},
```

Comment:
For readability both in code and on Pod specs, because not everyone is familiar with the shorthand form of Ceph flags, it's best to use the long form. This aids in comprehension and speeds up debugging.

```suggestion
		{Name: "CEPH_ARGS", Value: fmt.Sprintf("--rook-ceph-mon-host $(ROOK_CEPH_MON_HOST) --keyring %s", keyring.VolumeMount().KeyringFilePath())},
```

Reply:
@BlaineEXE I did the change but for whatever reason it doesn't work ... I tested it manually from the inside the pod and when I use `--ceph-mon-host`  the `radosgw-admin` fails to connect!

Reply:
@rkachach I think it is `--mon-host` only not `--ceph-mon-host`

Reply:
@subhamkrai that's correct, I re-tested the new args and they work :+1:

Reply:
great